### PR TITLE
Mvaliev 144 widgets library

### PR DESCRIPTION
**Pull request by user2589**

### CODE REVIEW

**Comment by mansuleman** [comment]
Shipt it

File: easy_phi/app.py [hunk 1]
```diff
@@ -80,6 +91,7 @@ def prepare(self):
         if err:
             self.set_status(400)
             self.finish({'error': err})
+            return
```
Comment (rezniknn) [inline easy_phi/app.py:94]: My pull request to dev contains this change already. Please make sure 'return' is not written twice when you will be merging with dev

Comment (user2589) [inline easy_phi/app.py:94]: it will cause merge conflict and I will have to update manually. In other words, it is not a problem

**Comment by rezniknn** [comment]
Ship it

**Pull request merged**